templates/plugin/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "fastify-tsconfig",
+    "compilerOptions": {
+      "outDir": "dist"
+    },
+    "include": [
```

Comment:
We don't need this include since this is a template for JS plugin that has also TS typings.

Reply:
I suppose we can leave this for eslint and standard.